unabridged/motion-demos
# Motion component list

# Description

This list demo is meant to show you a bug.

When an id is put on each item of a list around or directly inside a child component, and the list items can be deleted individually, I am getting ActionCable runtime errors.


## Testing

1. Pull this down, run the server and go to `/lists/`.
1. Explore various combinations of where/if to show the id and how the list should be generated by selecting the different buttons.
1. Try deleting records.  Have terminal open at the same time.  The list should tell you whether to expect runtime errors. 
1. Use the browser inspector tool to inspect the way the HTML is constructed for each option.


## Screenshots
<img width="841" alt="Screen Shot 2020-07-20 at 9 24 06 PM" src="https://user-images.githubusercontent.com/11672661/88005651-7947a180-cacf-11ea-8bfe-323842e92af4.png">

<img width="840" alt="Screen Shot 2020-07-20 at 9 27 21 PM" src="https://user-images.githubusercontent.com/11672661/88005818-d04d7680-cacf-11ea-9473-1c6a4c0bdf20.png">
